frontend/term.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <title>contained.af</title>
+
+        <link href="css/contained.min.css" media="all" rel="stylesheet" />
+    </head>
+    <body>
+        <div class="container">
+            <div class="page-header">
+                <h1>Container Escape Bounty CTF</h1>
+            </div>
+
+
+            <div id="console"></div>
+
+            <footer class="footer">
+                <div class="row">
+                    <div class="col-md-5">
+                        <p>There is a CTF on every VM instance. <b><em>If</em></b>&nbsp;
+                        you manage to break out of the container, email
+                        <a href="#">to-be-defined@example.com</a>
+                        and you will be rewarded. <br/>
+                        <em>If</em>&nbsp; you bother this email address with anything
+                        that is not the ascii art contents of the flag file you will be
+                        ignored.</p>
```

Comment:
The flag is currently not in ascii art :D

https://github.com/kinvolk/container-escape-bounty/blob/b2390f24c52de0f2356a7216e6382f4af28dc325/terraform/flagfiles.tf#L1

Reply:
removed it